test/http2-http2.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const h2url = require('h2url')
+const t = require('tap')
+const Fastify = require('fastify')
+const From = require('..')
+const fs = require('fs')
+const path = require('path')
+const certs = {
+  key: fs.readFileSync(path.join(__dirname, 'fixtures', 'fastify.key')),
+  cert: fs.readFileSync(path.join(__dirname, 'fixtures', 'fastify.cert'))
+}
```

Comment:
Can you add a test from http (non-secure) to http2 (non-secure)?

Reply:
Already there in [http-http2.js](https://github.com/fastify/fastify-reply-from/pull/6/files#diff-744c3c82c325d56497f72b3780a5ef4b) file